fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -645,6 +645,283 @@ T simulate(const T& in) {
 	return out;
 }
 
+ACTOR Future<std::string> getLayerStatus(Reference<ReadYourWritesTransaction> tr,
```

Comment:
Is this stuff supposed to be in this PR 'Add more trace events...'

Reply:
Whats a 'layer' in this context?

Reply:
These are copied from `backup_agent` code for debugging.